### PR TITLE
Update microsoft-power-bi-quiz.md

### PR DESCRIPTION
**Pull request by baker-jr-john**
- Changed "Bl" to "BI" in Q2, Q4, Q6, Q9, Q10, Q11, Q12, Q13, Q16, Q19, Q20, Q21, Q22, Q23
 - Selected answers for Q12, Q14, Q15, Q17, Q19, Q25, Q28, Q30
 - Added Q31 - Q42

### CODE REVIEW

File: microsoft-power-bi/microsoft-power-bi-quiz.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
  - [ ] B, C, D
  - [ ] D
 
-#### Q2. How do you create a chart visual in Power Bl Desktop Report view? (Select all that apply.)
+#### Q2. How do you create a chart visual in Power BI Desktop Report view? (Select all that apply.)
```
Comment (Ebazhanov) [inline microsoft-power-bi/microsoft-power-bi-quiz.md:15]: @baker-jr-john can you please explain why should we rename from BI to BL?

**Comment by Ebazhanov** [comment]
> * Changed "Bl" to "BI" in Q2, Q4, Q6, Q9, Q10, Q11, Q12, Q13, Q16, Q19, Q20, Q21, Q22, Q23
> * Selected answers for Q12, Q14, Q15, Q17, Q19, Q25, Q28, Q30
> * Added Q31 - Q42

@baker-jr-john thank you very clear PR, but I have a small question about renaming BI to BL

File: microsoft-power-bi/microsoft-power-bi-quiz.md (hunk 1 above)
Comment (baker-jr-john) [reply microsoft-power-bi/microsoft-power-bi-quiz.md:15]: I changed _from_ Bl (capital B, lower-case l) _to_ BI.

Comment (Ebazhanov) [reply microsoft-power-bi/microsoft-power-bi-quiz.md:15]: Nice one, OK good! Thank you again for your contribution will merge now!

**Comment by Ebazhanov** [comment]
@all-contributors please add @baker-jr-john for design, code

**Pull request merged**